platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/cache/CookieCache.java

```diff
@@ -124,32 +128,68 @@ public synchronized void setExpirationTime(int expirationMinutes) {
     }
 
     /**
-     * Listens for removal notifications from the cache and logs each time a removal is performed.
+     * Check to see if the provided Date/Time is outside of the SAML assertion time bounds
+     *
+     * @param key the corresponding key for the assertion
+     * @return true if the current time is outside of the SAML assertion time bounds
      */
-    private class RemovalListenerLogger implements RemovalListener<String, DataWrapper> {
-
-        @Override
-        public void onRemoval(RemovalNotification<String, DataWrapper> notification) {
-            LOGGER.debug("Expiring SAML ref:assertion {}:{} due to {}.",
-                    notification.getKey(),
-                    notification.getValue(),
-                    notification.getCause()
-                            .toString());
+    public boolean checkIfOutsideOfSSOTimeBounds(String key) {
```

Comment:
Methods that return a boolean and do not modify the state of the object are predicate methods. A useful convention is to ask a question in the method name. For example, "is assertion timestamp out of bounds?" could become isTimestampOutsideOfBounds().

Sometimes, even the predicate verb is unnecessary "assertionOutOfBounds". It is clear what a true/false means. 

Stylistically, I like to return true values for "happy path" and false values for "there is a problem". For example, name the method "assertionTimeIsValid". 

When the test gets moved to the SecurityAssertion interface/class, the name could be something like "isValidNow" or "isPresentlyValid"